versionedRouter.js

```diff
@@ -137,7 +135,17 @@ async function compareWithCdkV2(destType, input, flowType, v0Result) {
     ) {
       return;
     }
```

Comment:
As these changes are temporary to compare the performance, Do we want to get these changes merged to `master` or go with a custom branch ?

Reply:
nothing go wrong if these changes to in master, currently merging all these with master.